modules/special-rules/pages/weapon-mount-examples.adoc

```diff
@@ -0,0 +1,66 @@
+= Weapon mount examples
+
+.War engine weapon mounts and weapon arcs
+====
+Refer to the relevant data sheets and you will see that:
+
+* The Imperial Warlord Titan has four weapon mounts: Left Arm; Right Arm; Left Carapace; Right Carapace.
+* The Imperial Baneblade super-heavy tank has one weapon mount: Turret Guns.
+* The Imperial Shadowsword super-heavy has two weapon mounts: {death-ray}; Sponson Turrets.
+// TODO: It's not a 'Death Ray mount' — it's an 'X' ('Hull' or 'Volcano Cannon'?) mount that holds a Death Ray super-heavy weapon (or provides a Death Ray shot, depending on how we define mounts to cover the various cases). We need to decide what we're calling the mounts and then apply the appropriate name here. UPDATE: If my mount+arrays idea works, we can overhaul these examples to match.
+
+You will also see that each weapon mount has an icon to indicate its weapon arc.
+====
+
+.Firepower weapon mounts: Warhound Titan
+====
+The Warhound Titan has two weapon mounts, which use the same weapon arc.
+Each weapon mount holds a Heavy Weapon Battery that has a Firepower value of 6.
+
+[NOTE]
+=====
+In all cases, you may only allocate hits to units that are actually inside the weapon arc.
+
+{blank}[TODO: This is the implication I think, but I worry that it can be abused by the type of shameless player that deliberately makes sure that an arc is 'just so', to pick off a juicy unit that just happens to be poking out a bit farther than other closer units in the target detachment.]
+=====
```

Comment:
```suggestion
[NOTE]
=====
In all cases, you may only allocate hits to units that are actually inside the weapon arc.

{blank}[TODO: This is the implication I think, but I worry that it can be abused by the type of shameless player that deliberately makes sure that an arc is 'just so', to pick off a juicy unit that just happens to be poking out a bit farther than other closer units in the target detachment.]
=====
```

Shouldn't this be in the weapon arc rules, rather than in an example? ;-)

But yes, weapon arcs restrict line of fire and only units in line of fire can be hit without Artillery. So it's a LoF restriction rather than a weapon arc restriction.

Note that LoF-sniping is almost impossible to rule against, as you can also use the edges/corners of terrain to restrict LoF. But it's not much of an issue in E40k/Remastered as you would generally be wasting a bunch of Firepower/shots to do so, and the low-shot weapons where this would be a big issue in other games can already pick out individual targets.

Reply:
>Shouldn't this be in the weapon arc rules, rather than in an example? ;-)

Well yes, but having rewritten large fiddly bits in this and the main topic over and over again, fatigue was setting in. :-D I think that what prompted me at this point was that there was something about the example that felt like a potential gap for someone to argue due to the omission of this point... but as long as the point is correct, I'll look to fit it in better on the next pass.

>Note that LoF-sniping is almost impossible to rule against, as you can also use the edges/corners of terrain to restrict LoF. But it's not much of an issue in E40k/Remastered as you would generally be wasting a bunch of Firepower/shots to do so, and the low-shot weapons where this would be a big issue in other games can already pick out individual targets.

Okay, fair enough. It was just something that occurred to me as a corollary, but your points make sense of it as a non-issue.

Reply:
By the way, quick tip: You don't need to use the suggestion feature just to quote blocks like this — the conversation is bound to the lines you selected anyway and it says "Comment on lines +X to +Y".

Reply:
While I'm applying some other suggestions from conversations, I'll create this one to crop out my TODO note.

I'll get to the [NOTE] when I edit the topic again, see whether or not it should have a place. What I was concerned about is that this point might be easy to miss among all the other things in the examples that focus on the of targeting of detachments.

```suggestion
NOTE: In all cases, you may only allocate hits to units that are actually inside the weapon arc.
```